cloud/ec2_util.go

```diff
@@ -890,8 +897,9 @@ func associateIPAddressForHost(ctx context.Context, c AWSClient, h *host.Host) e
 	defer span.End()
 
 	assocAddrOut, err := c.AssociateAddress(ctx, &ec2.AssociateAddressInput{
-		InstanceId:   aws.String(h.Id),
-		AllocationId: aws.String(h.IPAllocationID),
+		InstanceId:         aws.String(h.Id),
+		AllocationId:       aws.String(h.IPAllocationID),
+		AllowReassociation: utility.TruePtr(),
```

Comment:
Can we update the function comment with a quick mention that it allows for address reassociation?

Reply:
Added.